documentation/1.9/content/samples/securityproviders-model.md

```diff
@@ -124,7 +124,7 @@ description: "Special handling and model semantics for WebLogic Server security
 
  Prior to using this tooling to create or update a domain with a custom security provider, there are several prerequisites.  First, WebLogic Server requires the custom MBean JAR to be in the Oracle Home directory before it can be configured, `WLSERVER/server/lib/mbeantypes`.  Second, WebLogic Scripting Tool, WLST, requires that the schema JAR be placed in the Oracle Home directory before WLST offline can be used to discover it or configure it, `ORACLEHOME/oracle_common/lib/schematypes`.  Generating an MBean JAR documentation can be found in the WebLogic Server [documentation](https://docs.oracle.com/en/middleware/standalone/weblogic-server/14.1.1.0/devsp/generate_mbeantype.html).  Generating the schema JAR can be done with the `prepareCustomProvider` script provided in the WebLogic Server installation.
 
- WebLogic allows you to define an alternate directory other than `WLSERVER/server/lib/mbeantypes` by using the system property `-Dweblogic.alternateTypesDirectory=dir`. In order for the custom provider jars to be loaded correctly by WLST when discovering or configuring a domain, set this system property in the `WLSDEPLOY_PROPERTIES` environment variable.
+ For the mbean jar, WebLogic allows you to define an alternate directory other than `WLSERVER/server/lib/mbeantypes` by using the system property `-Dweblogic.alternateTypesDirectory=dir`. For the WebLogic mbean schema type jar, you can use an alternate location by using -fmwconfig.alternateSchemaDirectory. In order for the custom provider jars to be loaded correctly by WLST when discovering or configuring a domain, set this system property in the `WLSDEPLOY_PROPERTIES` environment variable. Both of the properties take a comma separated list of paths to dirs containing the corresponding type of jar.
```

Comment:
I think "MBean" is correct - WLS docs and this page seem to use it.

Reply:
[oracle@96aea00aff4b oracle]$ oracle_common/common/bin/wlst.sh -fmwconfig.alternateSchemaDirectory

Initializing WebLogic Scripting Tool (WLST) ...

Jython scans all the jar files it can find at first startup. Depending on the system, this process may take a few minutes to complete, and WLST may not return a prompt right away.

Welcome to WebLogic Server Administration Scripting Shell

Type help() for help on available commands


The WebLogic Scripting Tool (WLST) is a command-line 
scripting interface that system administrators and 
operators use to monitor and manage WebLogic Server 
instances and domains.
 
Command line Options are: 
-i	: Runs the script and forces WLST in interactive mode. 
-help	: Prints this help message. 
-skipWLSModuleScanning	: Skips package scanning and caching for WLS modules. 
args	: Arguments can be a script file you would like to execute

Reply:
I think this is supposed to be something like:
```
CONFIG_JVM_ARGS="${CONFIG_JVM_ARGS} -Dfmwconfig.alternateSchemaDirectory=dir1[,dir2...]    -Dfmwconfig.alternateTypesDirectory=dir1[,dir2...]
```